setup.cfg

```diff
@@ -36,6 +36,8 @@ setup_requires =
 install_requires =
   pyvista>=0.30.1
   sunpy>=3.0.0
+  matplotlib>=3.1.0
+  scipy>=1.3.0
```

Comment:
Are these only required for the doc build?

Reply:
Yeah, I suppose

Reply:
If it is not required to install the package or even run the package, they should not be under this category.

Reply:
Yeah, I'll move it to docs.

Reply:
Failing with?

Reply:
We use `all_corner_coords_from_map`which required matplotlib as a dependency for map

Reply:
Ah I see. Then yeah, leave them there.